build.gradle

```diff
@@ -288,7 +288,7 @@ dependencies {
     implementation group: 'com.github.hmcts', name: 'document-management-client', version: '7.0.0'
     implementation group: 'com.github.hmcts', name: 'cmc-pdf-service-client', version: '7.0.1'
 
-    implementation group: 'com.github.hmcts', name: 'sscs-common', version: '4.21.18'
+    implementation group: 'com.github.hmcts', name: 'sscs-common', version: '1a3a4c8'
```

Comment:
This should not be changed. 4.21.18 is the version on master, which is my addition in 11092.

Reply:
Added to use the common changes which is not available in master.

Reply:
Someone have said on standup that there is a new commons version. We would need to update it here aswell

Reply:
Updated the common version with the latest changes.